pkg/gitreceive/k8s_util.go

```diff
@@ -222,3 +226,21 @@ func waitForPodCondition(c *client.Client, ns, podName string, condition func(po
 		return false, nil
 	})
 }
+
+func progress(msg string) chan bool {
+	tick := time.Tick(8000 * time.Millisecond)
+	quit := make(chan bool)
+	go func() {
+		for {
+			select {
+			case <-quit:
+				close(quit)
+				return
+			case <-tick:
+				fmt.Println(msg)
```

Comment:
can you print this to debug? above it looks like we're printing things like `waiting for the pod to come up...`, which is definitely debug-worthy